jekyll/_cci2/bundling-config.md

```diff
@@ -0,0 +1,905 @@
+---
+layout: classic-docs
+title: "Bundling Config"
```

Comment:
We should probably change this to "Reusable Config" or "Reusing Config" -- "Bundling" implies bundler for many people, but more importantly bundling is really an orbs thing rather than the commands/executors/parameters

Reply:
I _think_ I agree with Nathan here, but I'd like to take a step back and make sure I understand the purpose of this doc.

The work on "Orbs" added 3 groups of concepts into config:

A. "inline" jobs, executors and commands (which can define and take parameters)
B. "inline" orbs
C. registered orbs

(A) is mostly about re-using config within the same config.yml. (C) is for re-using config across projects, potentially across teams and companies with little collaboration. (B) is mostly to help people get to (C), but both (B) and (C) bundle or package up config that you _could_ express only in terms of (A).

With that said, is this doc meant to cover (A), (B), and (C) altogether? I think it's possible to introduce this topic to users starting from (A) only. That would be simpler to read through and digest, but do let me know if that level of separation is out of scope.